validator/src/main.rs

```diff
@@ -316,10 +319,18 @@ fn wait_for_restart_window(
                 "".to_string()
             } else {
                 format!(
-                    "| Full Snapshot Slot: {}",
+                    "| Full Snapshot Slot: {} | Incremental Snapshot Slot: {}",
```

Comment:
This is now printing a line exceeding terminal width, and apparently has broken progress-bar.
Instead of re-writing the same line, it is printing a new line each time.

Or maybe `42` below needs to be adjusted:
https://github.com/solana-labs/solana/blob/544a957ad/validator/src/lib.rs#L128

Reply:
@brooksprumo can we have this fixed?
`solana-validator monitor` is very annoying with this.

Reply:
Maybe the one showed in `monitor` is caused by https://github.com/solana-labs/solana/pull/19559
Either way this needs to be fixed.